OSDOCS-3203, pt.2: This PR reveals the ROSA UI documentation (#45954)
openshift/openshift-docs: modules/rosa-sts-account-wide-roles-and-policies.adoc

== Methods of account-wide role creation

You can create account-wide roles by using the `rosa` CLI tool or the {cluster-manager-url} guided installation. You can create the roles manually or by using an automatic process that uses pre-defined names for these roles and polices.

[discrete]
[id="rosa-sts-account-wide-roles-and-policies-creation-methods-manual_{context}"]
=== Manual ocm-role resource creation

You can use the manual creation method if you have the necessary CLI access to create these roles on your system. You can run this option in your desired CLI tool or from {cluster-manager}. After you start the manual creation process, the CLI presents a series of commands for you to run that create the roles and link them to the needed policies.

[discrete]
[id="rosa-sts-account-wide-roles-and-policies-creation-methods-auto_{context}"]
=== Automatic ocm-role resource creation

If you created an `ocm-role` resource with administrative permissions, you can use the automatic creation method in the CLI and from {cluster-manager}. Selecting this method creates the roles and policies that uses the default names.

> If you use the ROSA guided installation on {cluster-manager}, you must have created an `ocm-role` resource with administrative permissions in the first step of the guided cluster installation. Without this role, you cannot use the automatic Operator role and policy creation option.

Reviewer: ISG nit to avoid negative constructions:
```suggestion
If you use the ROSA guided installation on {cluster-manager}, you must have created an `ocm-role` resource with administrative permissions in the first step of the guided cluster installation. You must have this role to use the automatic Operator role and policy creation option.
```
